README.md

```diff
@@ -30,9 +30,14 @@ $ curl -u joel:joel1234 -sS http://localhost:5000/users -X GET
 Example from AngularJS:
 
 ```
+import { Http, Response, Headers } from '@angular/http'
+
```

Comment:
sorry @adrianq I don't see the format problem. Is is because of the "..." ? If that's the case I just wanted to represent that some code would be in the middle between the import and the use of the imported classes. Maybe I could have used [...] instead...or maybe that's not your point.